src/lib.rs

```diff
@@ -920,10 +925,40 @@ impl Octocrab {
                 }
             };
 
-            let result = request.send().await;
+            #[cfg(feature = "trace")]
+            let span = tracing::info_span!(
+                "github api call",
+                attempt=retries,
+                url=field::Empty,
+                service_name.name=field::Empty,
+                status=field::Empty,
+                error=field::Empty,
+            );
+            let result = if cfg!(feature = "trace") {
+                let service = self.base_url.host_str().unwrap_or(GITHUB_SERVICE);
+                span.record("service.name", service);
+                span.in_scope(|| {
+                    request.send()
+                }).await
+            } else {
+                request.send().await
```

Comment:
Shouldn't these `await`s be instrumented with the span? Otherwise it could be inaccurate I believe from the last time I read the docs.